java/semantickernel-core/src/main/java/com/microsoft/semantickernel/templateengine/blocks/TextBlock.java

```diff
@@ -6,8 +6,6 @@
 
 public class TextBlock extends Block implements TextRendering {
```

Comment:
This should be final and package protected

Reply:
we can make it final, but it is used outside the package